File: dotnet/src/shared/contracts/Shared.Contracts/Responses/PasswordService/EncryptedVaultResponse.cs
namespace Shared.Contracts.Responses.PasswordService
{
    public sealed record EncryptedVaultResponse(string Id, string Type, DateTime DateAdded, DateTime? DateUpdate, DateTime? DeletedAt, bool IsFavorite, bool IsArchive, bool IsInTrash, string EncryptedOverview, string EncryptedDetails, List<string> TagsIds, string IconId);
}

File: web/enigma-vault-ui/src/features/passwords/models/encrypted-vault.response.ts
export interface EncryptedVaultResponse {
    id: string; 
    type: string;
    dateAdded: Date;
    dateUpdate: Date | null;
    deletedAt: Date | null;
    isFavorite: boolean;
    isArchive: boolean; 
    isInTrash: boolean;
    encryptedOverview: string; 
    encryptedDetails: string, 
    tagsIds: string[]; 
    iconId: string;
}
